idl/sprot.idol

```diff
@@ -89,7 +89,7 @@ Interface(
             },
             reply : Result(
                 ok: "()",
-                err: CLike("UpdateError"),    // MsgError and UpdateError
+                err: CLike("SprotError"),    // SprotError and SprotError
```

Comment:
"SprotError and SprotError"?

Reply:
lol, nice automated replacement

Reply:
Fixed in [3a0f61a](https://github.com/oxidecomputer/hubris/pull/965/commits/3a0f61ac0031aaed46524561bf56eb41f20f085f)